example-app/database/seeders/DatabaseSeeder.php

```diff
@@ -1,32 +1,102 @@
 <?php
 
-namespace Database\Seeders;
+namespace Tests\Unit;
 
-use App\Models\{
-    Airline,
-    City,
-    Flight,
-};
-// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
-use Illuminate\Database\Seeder;
+use Tests\TestCase;
+use App\Models\Flight;
+use App\Models\Airline;
+use App\Models\City;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\JsonResponse;
 
-class DatabaseSeeder extends Seeder
+class FlightControllerTest extends TestCase
 {
+    use RefreshDatabase;
+
+    private $airline;
+    private $cities;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->seedDatabase();
+    }
+
+    protected function seedDatabase()
+    {
+        $this->cities = City::factory()->count(2)->create();
+
+        $this->airline = Airline::factory()->count(1)->create();
+
+        $this->airline->cities()->attach($this->cities[0]->id);
+        $this->airline->cities()->attach($this->cities[1]->id);
```

Comment:
You dont need to do this, it's already resolved by the Factory, have a look at this

https://laravel.com/docs/11.x/eloquent-factories#many-to-many-relationships